### PR TITLE
Update to function.php and readme

### LINKED ISSUE #2: readme.txt  – Installation instructions are wrong
```
== Installation ==

1. In your admin panel, go to Appearance > Themes and click the 'Add New' button.
2. Type in 'Hello Elementor Child' in the search form and hit the 'Enter' key on your keyboard.
3. Click on the 'Activate' button to use your new theme right away.
4. Navigate to Elementor and start building your site.
```

This simply isn't true / does not work right now, since the child theme is not part of the WordPress Repository yet.

### PR DESCRIPTION
**Pull request by pless84**
Fixes #2 and #3

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+ */
 
+/**
+ * Load child theme css and optional scripts
+ *
+ * @return void
+ */
 function hello_elementor_child_enqueue_scripts() {
 	wp_enqueue_style(
 		'hello-elementor-child',
```
Comment (KingYes) [inline functions.php]:
```suggestion
		'hello-elementor-child-style',
```

File: functions.php [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 		get_stylesheet_directory_uri() . '/style.css',
 		[
-			'hello-elementor'
+			'hello-elementor',
```
Comment (KingYes) [inline functions.php]:
```suggestion
			'hello-elementor-theme-style',
```

File: functions.php [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
 		],
 		'1.0.0'
 	);
 }
-add_action( 'wp_enqueue_scripts', 'hello_elementor_child_enqueue_scripts' );
+add_action( 'wp_enqueue_scripts', 'hello_elementor_child_enqueue_scripts', 99 );
```
Comment (KingYes) [inline functions.php]:
```suggestion
add_action( 'wp_enqueue_scripts', 'hello_elementor_child_enqueue_scripts' );
```

File: functions.php [hunk 4]
```diff
@@ -1,13 +1,23 @@
-<?php 
+<?php
+/**
+ * Theme functions and definitions
+ *
+ * @package HelloElementor
```
Comment (KingYes) [inline functions.php]:
```suggestion
 * @package HelloElementorChild
```

File: functions.php [hunk 5]
```diff
@@ -1,13 +1,23 @@
...
+ */
 
+/**
+ * Load child theme css and optional scripts
```
Comment (KingYes) [inline functions.php]:
```suggestion
 * Load child theme css and optional scripts.
```

**Comment by KingYes** [comment]
Please sign CLA, and we add some notes in your code. You need to accept them.

**Comment by KingYes** [comment]
Hey @pless84

File: functions.php (hunk 3 above)
Comment (pless84) [reply functions.php]: That goes against the solution, the priority is to fix #3

**Comment by KingYes** [comment]
@pless84 

Please add the last one..

**Comment by pless84** [comment]
> @pless84
> 
> Please add the last one..

I already commented that your change goes against fixing the issue

File: functions.php (hunk 3 above)
Comment (KingYes) [reply functions.php]: This will fix this: https://github.com/elementor/hello-theme-child/pull/5/commits/25a048901d16199a86a2fa71b4470a2cd85c833a

Comment (KingYes) [reply functions.php]: This will fix that issue: https://github.com/elementor/hello-theme-child/pull/5#discussion_r316047031

**Comment by pless84** [comment]
Cool should be all good now?

File: functions.php (hunk 2 above)
Comment (KingYes) [reply functions.php]: @pless84 

Change the dependency to fix the issue #3.

File: functions.php (hunk 3 above)
Comment (nuritsha) [reply functions.php]: @pless84,
Once you update the dependency array, there will be no need to fiddle with the priority.

**Comment by KingYes** [comment]
Thanks!

**Pull request merged**